Autopilot/Path Manager/MainBatterySensor.c

```diff
@@ -6,40 +6,34 @@
  */
 #include "main.h"
 #include "../Common/Common.h"
-#include "voltageSensor.h"
+#include "MainBatterySensor.h"
```

Comment:
Im not sure if this is good naming for the file. What makes this the main battery, and what differentiates from the other battery sensor? Like if Im reading the code, does main battery mean the battery attached to the picpilot powering the components, or the motors? Maybe we can come up with a clearer name here